QModManager/API/SMLHelper/MonoBehaviours/Fixer.cs

```diff
@@ -0,0 +1,55 @@
+namespace QModManager.API.SMLHelper.MonoBehaviours
+{
+    using UnityEngine;
+    using System.Reflection;
+    using Logger = QModManager.Utility.Logger;
+    using Harmony;
+
+    public class Fixer : MonoBehaviour, IProtoEventListener
+    {
+        [SerializeField]
+        public TechType techType;
+
+        [SerializeField]
+        public string ClassId;
+
+        private float time;
+        private bool initalized;
+
+        private static FieldInfo BuilderPrefab = AccessTools.Field(typeof(Builder), "prefab");
```

Comment:
Can this be accessed directly with the publicized assembly?

Reply:
It can, but we need its `FieldInfo`. If the `FieldInfo` is only used for `.GetValue`, we could replace this line.